File: iOS SoonZik/SoonZik/Models/Attachment.m
//
//  Attachment.m
//  SoonZik
//
//  Created by Maxime Sauvage on 12/09/2014.
//  Copyright (c) 2014 Coordina. All rights reserved.
//

#import "Attachment.h"

@implementation Attachment

- (id)initWithJsonObject:(NSDictionary *)json
{
    self = [super init];
    
    self.identifier = [[json objectForKey:@"id"] intValue];
    self.url = [json objectForKey:@"url"];
    self.contentType = [json objectForKey:@"content_type"];
    
    return self;
}

@end

File: Android/SoonZik/app/src/main/java/com/soonzik/soonzik/Attachment.java
package com.soonzik.soonzik;

import org.json.JSONObject;

/**
 * Created by kevin_000 on 21/03/2015.
 */
public class Attachment extends ActiveRecord {
    private int id = -1;
    private String url = "";
    private String file_size = "";
    private String content_type = "";

    public Attachment() {}

    public Attachment(JSONObject json) {
        super.createInstance(this, json, this.getClass());
    }

    @Override
    public String toString() {
        return (
                "id = " + Integer.toString(id)
                + " : url = " + url
                + " : file_size = " + file_size
                + " : content_type = " + content_type
                );
    }
}
